DeployDockerImage.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+APP_NAME=$1
+WORK_DIR=$2
+SATEL_DOCKER_USER=$3
+SATEL_DOCKER_PASS=$4
+CLIENT_DOCKER_USER=$5
+CLIENT_DOCKER_PASS=$6
+SATEL_REGISTRY=$7
+CLIENT_REGISTRY=$8
+DOCKERFILE=$9
+BRANCH_NAME=${10}
+TAG_NAME=${11}
+
+
+echo "APP_NAME=${APP_NAME}, WORK_DIR=${WORK_DIR}, SATEL_DOCKER_USER=${SATEL_DOCKER_USER}, SATEL_DOCKER_PASS=${SATEL_DOCKER_PASS}, CLIENT_DOCKER_USER=${CLIENT_DOCKER_USER}"
```

Comment:
Don't print out passwords into logs.
```suggestion
echo "APP_NAME=${APP_NAME}, WORK_DIR=${WORK_DIR}, SATEL_DOCKER_USER=${SATEL_DOCKER_USER}, CLIENT_DOCKER_USER=${CLIENT_DOCKER_USER}"
```

Reply:
Nope, I was using that to debug, but will remove them, it prints them as **** as well